### DISCUSSION TITLE
What do we think about adding in CC0 audio?

### DISCUSSION THREAD

**Discussion by TaylorAnderson** [NEW-CYLANDIA/little-warioware#13] [Ideas]
Specifically I found this from Calinou, bundling in a bunch of Kenney sounds for use in Godot https://github.com/Calinou/kenney-interface-sounds

**Comment by iznaut** [comment]
oooh i like this.

i think it would be cool to have some pre-packaged assets for ppl making microgames so folks that are more dev-focused can quickly prototype things and maybe there could be some sort of tagging for games that "need" art (so someone else could contact the game designer and offer to update the microgame with their own work)
